src/MultiButton.js

```diff
@@ -1,65 +1,88 @@
+import Tooltip from '@material-ui/core/Tooltip';
+import ListItemText from "@material-ui/core/ListItemText";
 
 class MultiButton extends React.Component {
   constructor(props){
     super(props)
     this.state = {
-      direction: 'right',
-      open: false,
+      anchorEl: null,
       audioPath: ''
     }
 
     this.text = 'MultiButton'
   }
   
   handleClose = () => {
-    this.setState({ open: false });
+    this.setState({ anchorEl: null });
   }
 
-  handleOpen = () => {
-    this.setState({ open: true });
-  }
-
-  handleSelect = (name) => {
-    alert(name)
+  handleOpen = (e) => {
+    this.setState({ anchorEl: e.currentTarget });
   }
 
   render() {
     return (
-      <SpeedDial
-        ariaLabel=""
-        hidden={false}
-        open={this.state.open}
-        direction={this.state.direction}
-        transitionDuration={0}
-        icon={<p>{this.props.text}</p>}
-
-        onMouseDown={this.handleOpen}
-        onMouseUp={() => {
-          this.props.onMouseUp(this.state.audioPath);
-          this.handleClose()
-        }}
-
-        // Play the default audio when clicking the main button.
-        onMouseEnter={
-          () => {this.setState(
-            {audioPath: (this.props.options.find((o) => o.default)).audioPath})
-          }
-        }
-      >
-        {this.props.options.map(option => (
-          <SpeedDialAction
-            icon={<p>{option.text}</p>}
-            classes={{button: 'multiButtonOption'}}
-            tooltipTitle={option.description}
-            delay={0}
-
-            onMouseEnter={() => {this.setState({audioPath: option.audioPath})}}
-          />
-        ))}
-      </SpeedDial>
+      <Grid item key={this.props.key}>
+        <Button
+          className="MultiButton-main"
+          aria-controls="customized-menu"
+          aria-haspopup="true"
+          variant="contained"
+          color="primary"
+          onMouseDown={this.handleOpen}
+          onMouseUp={() => {
+            this.props.onMouseUp(this.state.audioPath);
+            this.handleClose()
+          }}
+          onMouseEnter={
+            () => {this.setState(
+              {audioPath: (this.props.options.find((o) => o.default)).audioPath})
+          }}
+        >
+          {this.props.text}
+        </Button>
+        <Menu
+          className="MultiButton-menu"
+          anchorEl={this.state.anchorEl}
+          keepMounted={false}
+          open={Boolean(this.state.anchorEl)}
+          onClose={this.handleClose}
+          elevation={4}
+          getContentAnchorEl={null}
+          autoFocus={false}
+          transitionDuration={0}
+          variant="menu"
+          anchorOrigin={{
+            vertical: "bottom",
+            horizontal: "right"
+          }}
+          transformOrigin={{
+            vertical: "top",
+            horizontal: "right"
+          }}
+        >
+          {this.props.options.map(option => (
+            <Tooltip title={option.description} placement='bottom'>
```

Comment:
This is a little nitpicky but it's confusing to have `text` and `description` on the object. Whenever you add these to `DemoLayout` it would make more sense to have `text` be something like `display_name` or `short_name` just to make it clear that that's the shortened name we see on the button vs the tooltip text (I think `description` is a good name for that but you could also make it `tooltip_text` or something).